free-programming-books.md

```diff
@@ -1287,8 +1288,10 @@ Kerridge (PDF) (email address *requested*, not required)
 ### Flutter
 
 * [Cookbook](https://flutter.dev/docs/cookbook)
+* [Flutter Complete Reference](https://fluttercompletereference.com/preview) - Alberto Miola (PDF) (Free only introduction & first chapter)
+* [Flutter Libraries We Love](https://codemagic.io/flutter-libraries-ebook) - CodeMagic.io (email address *requested*)
```

Comment:
this site requires a working email, not just requests it. Our bar is much higher for such sites, in most cases we won't llist them.